turbo/cli/flags.go

```diff
@@ -55,6 +55,11 @@ var (
 		Usage: "enable experimental downloader v2",
 	}
 
+	PruningFlag = cli.BoolFlag{
+		Name:  "prune",
+		Usage: "Enable pruning ancient data",
```

Comment:
Add comment that this feature experimental yet. If pruning it’s storage mode, why it’s not part of —storage-mode flag?

Reply:
Mostly for reasons of aesthetics and discoverability. Pruning is not _just_ another optional data item - it's a substantially different sync mode. Also, this flag will be set _much more often_ than other storage mode toggles.

I only put pruning inside storage mode to reuse the code around checking mode compatibility.